web-app/js/portal/filter/ui/DateFilterPanel.js

```diff
@@ -120,26 +120,16 @@ Portal.filter.ui.DateFilterPanel = Ext.extend(Portal.filter.ui.BaseFilterPanel,
 
     _applyDateFilterPanel: function(component) {
```

Comment:
Perhaps we don't need to use the passed component variable either - just refer to the panel - that would make the code below even clearer without having to refer to where the code is called